lib/gnat/aunit_shared.gpr

```diff
@@ -9,7 +9,8 @@ project AUnit_Shared is
        "zfp-cross",      -- used for zfp runtimes on some cross ports
        "ravenscar",      -- used for full ravenscar runtimes
        "ravenscar-cert", -- used for ravenscar-cert runtimes
-       "cert"            -- used for cert runtimes
+       "cert",           -- used for cert runtimes
+       "ravenscar-sfp"   -- used for ravenscar-sfp for bareboard
```

Comment:
What I meant is that you don't need this new value, as soon as you can specify -XAUNIT_RUNTIME=zfp for bare metal ravenscar-sfp runtimes.

Reply:
Tbh I think it's better to have the runtime name specified here instead of adding some special case somewhere in anod to replace ravenscar-sfp with zfp. This increases the complexity of the anod code and the effort to debug it in case of an error while I can't see this downside with the change done here.

Reply:
FWIW, Johannes' proposal seems cleaner to me too, and it's pretty localized as well. And I think I would prefer the AUnit users to use -XAUNIT_RUNTIME values that actually matches the runtime they are using.

Reply:
I've added Vasiliy to help us review the change. I suspect based on other discussions with Vasiliy that he might agree with Jerome.